src/mpid/ch4/netmod/ofi/ofi_coll_params.h

```diff
@@ -15,11 +15,11 @@ typedef union {
 typedef enum {
     MPIDI_OFI_Bcast_intra_binomial_id,
     MPIDI_OFI_Bcast_intra_scatter_recursive_doubling_allgather_id,
-    MPIDI_OFI_Bcast_intra_scatter_ring_allgather_id
+    MPIDI_OFI_Bcast_intra_scatter_ring_allgather_id,
```

Comment:
This is a code formatting only change and should be a separate patch (or dropped).